### DISCUSSION TITLE
Collection import deletes Users collection

### DISCUSSION THREAD

**Discussion by rawestmoreland** [pocketbase/pocketbase#1082] [Q&A] [answered]
When I attempt to import collection JSON from an export, it seems as though the users collection will be deleted since it isn't included in the JSON.

This is v0.8.0

<img width="931" alt="Screenshot 2022-11-21 at 8 54 12 AM" src="https://user-images.githubusercontent.com/30242837/203113873-7dea6b47-532e-41b2-8ad8-45858c24bb64.png">

**Comment by ganigeorgiev** [comment, marked as answer]
Is the json you are trying to import generated from older PocketBase version?

Based on the screenshot it seems that you are trying to import a v0.7.x schema into v0.8.0 and this will not work and you'll have to adjust the json to reflect the [changes in v0.8](https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#v080) (or first migrate the environment from where you got the json and then export it again).

**Reply by rawestmoreland** [reply]
Got it! Thanks for the help!
